src/InstrumentationEngine/ProfilerManager.h

```diff
@@ -398,8 +410,9 @@ namespace MicrosoftInstrumentationEngine
         HRESULT AssemblyUnloadStartedImpl(_In_ AssemblyID assemblyId);
         HRESULT AssemblyUnloadFinishedImpl(_In_ AssemblyID assemblyId, _In_ HRESULT hrStatus);
 
-        HRESULT SetupInstrumentationMethods(
-            _In_ const std::vector<CComPtr<CConfigurationSource>>& configSources
+        HRESULT InvokeThreadRoutine(
+            _In_ const std::vector<CComPtr<CConfigurationSource>>& configSources,
```

Comment:
Remove this parameter. If the caller is actually providing configuration sources, let it set m_configSources before invoking the thread routine.